Tool/PlatformList.csv

```diff
@@ -0,0 +1,6 @@
+SystemAbbreviation,SystemName
```

Comment:
Remove these system names and put generic "System A," System B" for the github repo.

Reply:
Removed systems and added generic system names.